Sources/SwiftSentry/SentrySDK.swift

```diff
@@ -49,9 +49,14 @@ public enum SentrySDK {
             fatalError("Unable to find caches directory for storing Sentry reports!")
         }
 
+        // Use the dsn + environment to create some variability in the hash
+        // to put the same app running in different environments in different
+        // places on disk to avoid any potential contention.
+        let applicationHash = "\(options.dsn)\(options.environment)".hash
```

Comment:
Hmm, any reason to prefer this over say:

```swift
var hasher: Hasher = Hasher()
[options.dns, options.environment].hash(into: &hasher)
String(haher.finalize())
```

Reply:
Not really other than it's fewer lines of code. Will the hasher do something fundamentally different here? Looking at these in a REPL they seem to spit out different values (obvi), but both seem suitable 

![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/yo50SV5KO2cmCHVizzqY/108c29ac-76d1-4ff1-90c2-5ed2213dea61.png)

Reply:
In my mind the "more code" approach is more semantically "hash these items into a single hash" where as the "less code" approach is saying, "hash this string". The semantics of the former in my mind say more clearly that I want to base the hash on these items (and while not exactly that, the idea is that you are saying hash this `Set` rather than `Array`, but seems silly to create a `Set` just to say that). I think that the less code version is fine, but should get a comment saying that and indicate that the ordering is arbitrarily chosen.

Reply:
Decided to use the hasher, since after thinking about it I agree with you!